flavors/template-Exasol-all-python-3.8-cuda-conda/flavor_base/language_deps/packages/conda_packages

```diff
@@ -1,7 +1,7 @@
 libxcrypt|4.4.36 # Needed for building the udfclient and because libcrypt got removed from conda sysroot, see https://github.com/conda-forge/linux-sysroot-feedstock/issues/52
 python|3.8.13
 numpy|1.24.3
-pandas|2.0.2
+pandas|2.0.3
```

Comment:
Why here Pandas 2.0.3 and in the normal template container 2.0.2

Reply:
Update pandas also on the normal template flavor.